horreum-backend/src/main/java/io/hyperfoil/tools/horreum/datastore/CollectorApiDatastore.java

```diff
@@ -0,0 +1,105 @@
+package io.hyperfoil.tools.horreum.datastore;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.hyperfoil.tools.horreum.api.data.datastore.DatastoreType;
+import io.hyperfoil.tools.horreum.api.data.datastore.CollectorApiDatastoreConfig;
+import io.hyperfoil.tools.horreum.entity.backend.DatastoreConfigDAO;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.BadRequestException;
+import org.jboss.logging.Logger;
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Optional;
+
+@ApplicationScoped
+public class CollectorApiDatastore implements Datastore {
+
+    protected static final Logger log = Logger.getLogger(CollectorApiDatastore.class);
+
+    @Inject
+    ObjectMapper mapper;
+
+    @Override
+    public DatastoreResponse handleRun(JsonNode payload,
+                              JsonNode metaData,
+                              DatastoreConfigDAO configuration,
+                              Optional<String> schemaUriOptional,
+                              ObjectMapper mapper)
+            throws BadRequestException{
+
+        try {
+
+            if ( metaData != null ){
+                log.warn("Empty request: " + metaData.toString());
+                throw new BadRequestException("Empty request: " + metaData);
+            }
+            metaData = payload;
+
+            final CollectorApiDatastoreConfig jsonDatastoreConfig = mapper.treeToValue(configuration.configuration, CollectorApiDatastoreConfig.class);
+
+            if ( jsonDatastoreConfig != null ){
+
+                HttpClient client = HttpClient.newHttpClient();
+                HttpRequest.Builder builder = HttpRequest.newBuilder()
+                        .uri(java.net.URI.create(jsonDatastoreConfig.url + "/api/v1/image-stats/tag/" + payload.get("tag").asText()));
```

Comment:
should the url `path` be a separate config param, or part of the `jsonDatastoreConfig.url`, if the api changes (e.g. a version bump) then this hard-coded path will likely be no longer valid